src/components/styled/StyledNew.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+export const StyledHeadbanner = styled.div`
+	padding: 22px 20px;
+	background-color: #0bda51;
+	font-size: x-large;
+	text-align: center;
+`;
+
+export const StyledNavbar = styled.li`
```

Comment:
the Neame indicates that there is a `nav`element , but you get a List item. Thats not good naming or bad semantic. please check what do you need here and name it accordingly :)